docs/content/latest/api/ysql/exprs/window_functions/analyzing-a-normal-distribution/cr-do-cume-dist.md

```diff
@@ -2,7 +2,7 @@
 title: cr_do_cume_dist.sql
 linkTitle: cr_do_cume_dist.sql
 headerTitle: cr_do_cume_dist.sql
-description: Create the function that creates the histogram output.
+description: Part of the code kit for the "Analyzing a normal distribution" section within the YSQL window fucntions documentation.
```

Comment:
Same issue here. Each description should give a reason to differentiate the page from the other ones.

Reply:
Fixed each of these seventeen files using the scheme we agreed. The description text starts with the name of the `.sql` file as which that you're told to save the script. Then comes the identical _"The bucket allocation scheme"_ boilerplate.